packages/vm/benches/main.rs

```diff
@@ -169,16 +180,25 @@ fn bench_cache(c: &mut Criterion) {
         });
     });
 
-    group.bench_function("analyze", |b| {
+    let path_iter = glob::glob("testdata/*.wasm").unwrap().flat_map(Result::ok);
+    for contract_path in path_iter {
+        let contract_name = contract_path.file_name().unwrap().to_str().unwrap();
+        if !BENCH_CONTRACTS.contains(&contract_name) {
+            continue;
+        }
```

Comment:
Is there a reason to use an allowlist instead of just iterating `BENCH_CONTRACTS` directly and building the path ourselves?

Reply:
Nope, gonna get rid of the glob. How this evolved was just, I first tried to just glob all of the `.wasm` files from the test data but obviously some have incompatibilities, so yeah, then the allowlist came.

Gonna remove the allowlist and just interpolate the path manually.